.gitignore

```diff
@@ -2,3 +2,4 @@
 .tox
 dist/
 wideq_state.json
+wideq/__pycache__
```

Comment:
Maybe just `__pycache__`? We can ignore it wherever it appears.

Reply:
updated